crawler/containers.py

```diff
@@ -47,16 +47,15 @@ def get_filtered_list_of_containers(
     The partitioning is given by `partition_strategy`.
     """
 
-    environment = options.get('environment', defaults.DEFAULT_ENVIRONMENT)
+    environment = options.get('environment',
+                              get_config()['general']['environment'])
```

Comment:
yeah, i'm eventually moving all the config loading to be done at init time only. It's just that with this PR wanted to start by just replacing defaults.py with the new location of defaults.